README.md

```diff
@@ -17,10 +17,11 @@ Prerequisites
 
 This repository contains connectors as following:
 
-|          Connector          | OceanBase Compatible Mode |               Supported Features                |
-|-----------------------------|---------------------------|-------------------------------------------------|
-| Flink Connector: OceanBase  | MySQL, Oracle             | [Sink](docs/sink/flink-connector-oceanbase.md)  |
-| Flink Connector: OBKV HBase | OBKV HBase                | [Sink](docs/sink/flink-connector-obkv-hbase.md) |
+|               Connector                | OceanBase Compatible Mode |                    Supported Features                     |
+|----------------------------------------|---------------------------|-----------------------------------------------------------|
+| Flink Connector: OceanBase             | MySQL, Oracle             | [Sink](docs/sink/flink-connector-oceanbase.md)            |
+| Flink Connector: OceanBase Direct Load | MySQL                     | [Sink](docs/sink/flink-connector-oceanbase-directload.md) |
```

Comment:
It is not accurate to say that it works for mysql mode, obkv-hase and obkv-redis mode of new 4.x versions can also use direct load. You can remove the compatible mode column and describe the features details in the 'Supported Features' column.

Reply:
Fixed.